preso-ro/preso_ro.rb

```diff
@@ -0,0 +1,64 @@
+require 'erb'
+require 'date'
+require 'chronic'
+
+
+require_relative "./poros/sponsor"
+require_relative "./poros/speaker"
+require_relative "./poros/presentation"
+require_relative "./poros/company"
+
+class PresoGo
+
+  def self.generate(path, pres_prefix)
```

Comment:
This could certainly do with some refactoring, so just squint if you get this far.